src/pswap.md

```diff
@@ -30,4 +41,4 @@ Additionally, special rewards to strategic purchasers of XOR during the special
 
 **Example:** Let's assume that buyback and burn from transaction fees for a day resulted in 10,000 PSWAP being burned. If the daily reward is 80%, then 8,000 new PSWAP tokens are given out proportionally across all liquidity providers and 1,000 PSWAP tokens are given to the SORA Parliament and 1,000 PSWAP are distributed between the participants of the bonus strategic programs
 
-Since more PSWAP tokens are always burned than minted, PSWAP tokens are deflationary by nature. 600,000,000+ PSWAP tokens were created in the genesis event. PSWAP future token value is not dependent on the effort of the Polkaswap or Soramitsu team. In the future new PSWAP tokens may be minted by governance decisions of the SORA Parliament.
+Since more PSWAP tokens are always burned than minted, PSWAP tokens are deflationary by nature. 600,000,000+ PSWAP tokens were created in the genesis event. PSWAP future token value is not dependent on the effort of the Polkaswap or SORAmitsu team. In the future new PSWAP tokens may be minted by governance decisions of the SORA Parliament.
```

Comment:
```suggestion
Since more PSWAP tokens are always burned than minted, PSWAP tokens are deflationary by nature. 600,000,000+ PSWAP tokens were created in the genesis event. PSWAP future token value is not dependent on the effort of the Polkaswap or the SORAMITSU team. In the future new PSWAP tokens may be minted by governance decisions of the SORA Parliament.
```